Client/Frontend/Toolbar+URLBar/TabToolbarHelper.swift

```diff
@@ -68,20 +69,28 @@ open class TabToolbarHelper: NSObject {
             middleButtonState = .search
             toolbar.multiStateButton.setImage(ImageSearch, for: .normal)
             toolbar.multiStateButton.accessibilityLabel = .TabToolbarSearchAccessibilityLabel
+            toolbar.multiStateButton.largeContentTitle = .TabToolbarSearchAccessibilityLabel
+            toolbar.multiStateButton.largeContentImage = ImageSearch
             toolbar.multiStateButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.searchButton
         case (.reload, .pad):
             middleButtonState = .reload
             toolbar.multiStateButton.setImage(ImageReload, for: .normal)
             toolbar.multiStateButton.accessibilityLabel = .TabToolbarReloadAccessibilityLabel
+            toolbar.multiStateButton.largeContentTitle = .TabToolbarReloadAccessibilityLabel
+            toolbar.multiStateButton.largeContentImage = ImageReload
             toolbar.multiStateButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.reloadButton
         case (.stop, .pad):
             middleButtonState = .stop
             toolbar.multiStateButton.setImage(ImageStop, for: .normal)
             toolbar.multiStateButton.accessibilityLabel = .TabToolbarStopAccessibilityLabel
+            toolbar.multiStateButton.largeContentTitle = .TabToolbarStopAccessibilityLabel
+            toolbar.multiStateButton.largeContentImage = ImageStop
             toolbar.multiStateButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.stopButton
         default:
             toolbar.multiStateButton.setImage(ImageHome, for: .normal)
             toolbar.multiStateButton.accessibilityLabel = .TabToolbarHomeAccessibilityLabel
+            toolbar.multiStateButton.largeContentImage = ImageHome
+            toolbar.multiStateButton.largeContentTitle = .TabToolbarHomeAccessibilityLabel
```

Comment:
I reused existing accessibility labels. Should I create a separate localized strings for the `largeContentTitle`s?